modules/apps/petra/petra-log4j/src/main/java/com/liferay/petra/log4j/Log4JUtil.java

```diff
@@ -102,6 +105,19 @@ public static void configureLog4J(URL url) {
 		return new HashMap<>(_customLogSettings);
 	}
 
+	public static List<Log4JLevelConfig> getLog4JLevelConfigs() {
```

Comment:
@marianoalvarosaiz Please do not modify Log4jUtil, move all the logic of this method to where it is used. Log4jUtil should already export all the methods you need.

Thanks.
Tina.